2022-03-08 00:56:51,281 INFO    MainThread:3014 [wandb_setup.py:_flush():75] Loading settings from /home/lieberze/.config/wandb/settings
2022-03-08 00:56:51,281 INFO    MainThread:3014 [wandb_setup.py:_flush():75] Loading settings from /home/lieberze/DP/Thesis/06_adapter/wandb/settings
2022-03-08 00:56:51,282 INFO    MainThread:3014 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-03-08 00:56:51,282 INFO    MainThread:3014 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-03-08 00:56:51,282 INFO    MainThread:3014 [wandb_init.py:_log_setup():386] Logging user logs to /home/lieberze/DP/Thesis/06_adapter/wandb/run-20220308_005651-ho649879/logs/debug.log
2022-03-08 00:56:51,282 INFO    MainThread:3014 [wandb_init.py:_log_setup():387] Logging internal logs to /home/lieberze/DP/Thesis/06_adapter/wandb/run-20220308_005651-ho649879/logs/debug-internal.log
2022-03-08 00:56:51,283 INFO    MainThread:3014 [wandb_init.py:init():420] calling init triggers
2022-03-08 00:56:51,283 INFO    MainThread:3014 [wandb_init.py:init():427] wandb.init called with sweep_config: {}
config: {}
2022-03-08 00:56:51,283 INFO    MainThread:3014 [wandb_init.py:init():442] re-initializing run, found existing run on stack: 3o4t6k2k
2022-03-08 00:56:51,289 INFO    MainThread:3014 [wandb_run.py:finish():1468] finishing run liebelife/adapter_CV_3x3/3o4t6k2k
2022-03-08 00:56:51,290 INFO    MainThread:3014 [jupyter.py:save_history():429] not saving jupyter history
2022-03-08 00:56:51,290 INFO    MainThread:3014 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-03-08 00:56:51,290 INFO    MainThread:3014 [wandb_init.py:_jupyter_teardown():318] cleaning up jupyter logic
2022-03-08 00:56:51,290 INFO    MainThread:3014 [wandb_run.py:_atexit_cleanup():1797] got exitcode: 0
2022-03-08 00:56:51,292 INFO    MainThread:3014 [wandb_run.py:_restore():1769] restore
2022-03-08 00:56:52,177 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 880
  total_bytes: 880
}

2022-03-08 00:56:52,331 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 880
  total_bytes: 880
}

2022-03-08 00:56:52,541 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 880
  total_bytes: 880
}

2022-03-08 00:56:53,169 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 880
  total_bytes: 12078
}

2022-03-08 00:56:53,273 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 880
  total_bytes: 12078
}

2022-03-08 00:56:53,388 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 880
  total_bytes: 12078
}

2022-03-08 00:56:53,490 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 12078
  total_bytes: 12078
}

2022-03-08 00:56:53,595 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 12078
  total_bytes: 12078
}

2022-03-08 00:56:53,697 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 12078
  total_bytes: 12078
}

2022-03-08 00:56:53,800 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 12078
  total_bytes: 12078
}

2022-03-08 00:56:53,902 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 12078
  total_bytes: 12078
}

2022-03-08 00:56:54,422 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 12078
  total_bytes: 12078
}

2022-03-08 00:56:54,835 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 12078
  total_bytes: 12078
}
local_info {
}

2022-03-08 00:56:56,286 INFO    MainThread:3014 [wandb_run.py:_append_files():2194] logging synced files
2022-03-08 00:56:56,295 INFO    MainThread:3014 [wandb_init.py:init():471] starting backend
2022-03-08 00:56:56,295 INFO    MainThread:3014 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-03-08 00:56:56,298 INFO    MainThread:3014 [backend.py:ensure_launched():219] starting backend process...
2022-03-08 00:56:56,406 INFO    MainThread:3014 [backend.py:ensure_launched():225] started backend process with pid: 5062
2022-03-08 00:56:56,407 INFO    MainThread:3014 [wandb_init.py:init():480] backend started and connected
2022-03-08 00:56:56,446 INFO    MainThread:3014 [wandb_run.py:_label_probe_notebook():921] probe notebook
2022-03-08 00:56:56,446 INFO    MainThread:3014 [wandb_run.py:_label_probe_notebook():931] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-03-08 00:56:56,446 INFO    MainThread:3014 [wandb_init.py:init():550] updated telemetry
2022-03-08 00:56:56,648 INFO    MainThread:3014 [wandb_init.py:init():581] communicating current version
2022-03-08 00:56:58,142 INFO    MainThread:3014 [wandb_init.py:init():586] got version response upgrade_message: "wandb version 0.12.11 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-03-08 00:56:58,144 INFO    MainThread:3014 [wandb_init.py:init():596] communicating run to backend with 30 second timeout
2022-03-08 00:56:58,353 INFO    MainThread:3014 [wandb_init.py:init():624] starting run threads in backend
2022-03-08 00:57:03,365 INFO    MainThread:3014 [wandb_run.py:_console_start():1827] atexit reg
2022-03-08 00:57:03,367 INFO    MainThread:3014 [wandb_run.py:_redirect():1701] redirect: SettingsConsole.WRAP
2022-03-08 00:57:03,367 INFO    MainThread:3014 [wandb_run.py:_redirect():1738] Wrapping output streams.
2022-03-08 00:57:03,368 INFO    MainThread:3014 [wandb_run.py:_redirect():1762] Redirects installed.
2022-03-08 00:57:03,368 INFO    MainThread:3014 [wandb_init.py:init():651] run started, returning control to user process
2022-03-08 00:57:36,667 INFO    MainThread:3014 [wandb_setup.py:_flush():75] Loading settings from /home/lieberze/.config/wandb/settings
2022-03-08 00:57:36,667 INFO    MainThread:3014 [wandb_setup.py:_flush():75] Loading settings from /home/lieberze/DP/Thesis/06_adapter/wandb/settings
2022-03-08 00:57:36,667 INFO    MainThread:3014 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-03-08 00:57:36,667 INFO    MainThread:3014 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-03-08 00:57:36,667 INFO    MainThread:3014 [wandb_init.py:_log_setup():386] Logging user logs to /home/lieberze/DP/Thesis/06_adapter/wandb/run-20220308_005736-1sguien4/logs/debug.log
2022-03-08 00:57:36,668 INFO    MainThread:3014 [wandb_init.py:_log_setup():387] Logging internal logs to /home/lieberze/DP/Thesis/06_adapter/wandb/run-20220308_005736-1sguien4/logs/debug-internal.log
2022-03-08 00:57:36,668 INFO    MainThread:3014 [wandb_init.py:_jupyter_setup():336] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7fae1a5d2978>
2022-03-08 00:57:36,668 INFO    MainThread:3014 [wandb_init.py:init():420] calling init triggers
2022-03-08 00:57:36,668 INFO    MainThread:3014 [wandb_init.py:init():427] wandb.init called with sweep_config: {}
config: {}
2022-03-08 00:57:36,669 INFO    MainThread:3014 [wandb_init.py:init():442] re-initializing run, found existing run on stack: ho649879
2022-03-08 00:57:36,671 INFO    MainThread:3014 [wandb_run.py:finish():1468] finishing run liebelife/adapter_CV_3x3/ho649879
2022-03-08 00:57:36,671 INFO    MainThread:3014 [wandb_run.py:_atexit_cleanup():1797] got exitcode: 0
2022-03-08 00:57:36,673 INFO    MainThread:3014 [wandb_run.py:_restore():1769] restore
2022-03-08 00:57:37,452 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 880
  total_bytes: 880
}

2022-03-08 00:57:37,595 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 880
  total_bytes: 880
}

2022-03-08 00:57:37,859 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 880
  total_bytes: 880
}

2022-03-08 00:57:38,420 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 880
  total_bytes: 21763
}

2022-03-08 00:57:38,524 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 880
  total_bytes: 21763
}

2022-03-08 00:57:38,634 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 880
  total_bytes: 21763
}

2022-03-08 00:57:38,737 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 13339
  total_bytes: 21763
}

2022-03-08 00:57:38,842 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 21763
  total_bytes: 21763
}

2022-03-08 00:57:38,946 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 21763
  total_bytes: 21763
}

2022-03-08 00:57:39,049 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 21763
  total_bytes: 21763
}

2022-03-08 00:57:39,152 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 21763
  total_bytes: 21763
}

2022-03-08 00:57:39,254 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 21763
  total_bytes: 21763
}

2022-03-08 00:57:39,357 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 21763
  total_bytes: 21763
}

2022-03-08 00:57:39,922 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 21763
  total_bytes: 21763
}

2022-03-08 00:57:40,443 INFO    MainThread:3014 [wandb_run.py:_wait_for_finish():1929] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 21763
  total_bytes: 21763
}
local_info {
}

2022-03-08 00:57:41,886 INFO    MainThread:3014 [wandb_run.py:_append_files():2194] logging synced files
